### ISSUE TITLE
[Chromecast] Random crash during playback

### ISSUE THREAD

**Issue by gitdeath** [jellyfin/jellyfin-androidtv#2322]
### Describe the bug

1. Playing a show
2. Randomly (feels like) crash

### Logs

---
client: Jellyfin for Android TV
client_version: 0.14.5
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException: Attempt to invoke virtual method 'org.jellyfin.apiclient.model.dto.BaseItemType org.jellyfin.apiclient.model.dto.BaseItemDto.getBaseItemType()' on a null object reference
	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity$2.run(FullDetailsActivity.java:256)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7839)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

```  
***Logcat***: 
```log
--------- beginning of main
12-06 21:13:28.297 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 248 (pos 1378012 - event end 1377709)
12-06 21:13:30.481 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:30.638 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:31.324 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 249 (pos 1381036 start 1380587 end 1382672)
12-06 21:13:33.340 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 249 (pos 1383052 - event end 1382672)
12-06 21:13:33.639 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:33.684 D/Volley  (21374): [205] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1938> [lifetime=3045], [size=0], [rc=204], [retryCount=0]
12-06 21:13:33.685 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:34.857 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 250 (pos 1384568 start 1384216 end 1385842)
12-06 21:13:36.019 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:36.382 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 250 (pos 1386099 - event end 1385842)
12-06 21:13:36.643 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:37.391 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 251 (pos 1387107 start 1386885 end 1388136)
12-06 21:13:38.905 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 251 (pos 1388620 - event end 1388136)
12-06 21:13:39.645 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:39.690 D/Volley  (21374): [206] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1940> [lifetime=3046], [size=0], [rc=204], [retryCount=0]
12-06 21:13:39.691 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:40.924 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 252 (pos 1390637 start 1390472 end 1393850)
12-06 21:13:42.646 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:42.964 D/Volley  (21374): [207] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1941> [lifetime=3319], [size=0], [rc=204], [retryCount=0]
12-06 21:13:42.965 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:44.454 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 252 (pos 1394171 - event end 1393850)
12-06 21:13:44.959 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 253 (pos 1394667 start 1394559 end 1396645)
12-06 21:13:45.649 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:45.840 D/Volley  (21374): [205] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1942> [lifetime=3192], [size=0], [rc=204], [retryCount=0]
12-06 21:13:45.840 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:46.977 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 253 (pos 1396694 - event end 1396645)
12-06 21:13:47.483 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 254 (pos 1397200 start 1396728 end 1399106)
12-06 21:13:48.651 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:49.219 D/Volley  (21374): [208] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1943> [lifetime=3570], [size=0], [rc=204], [retryCount=0]
12-06 21:13:49.220 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:49.501 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 254 (pos 1399210 - event end 1399106)
12-06 21:13:51.654 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:52.263 D/Volley  (21374): [206] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1944> [lifetime=3610], [size=0], [rc=204], [retryCount=0]
12-06 21:13:52.265 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:54.655 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:56.142 D/Volley  (21374): [207] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1945> [lifetime=4488], [size=0], [rc=204], [retryCount=0]
12-06 21:13:56.143 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:57.067 D/CustomPlaybackOverlayFragment(21374): rendering subtitle event: 255 (pos 1406776 start 1406404 end 1408448)
12-06 21:13:57.657 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:58.743 D/Volley  (21374): [205] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] http://192.168.1.200:8096/Sessions/Playing/Progress 0x85bc1196 NORMAL 1946> [lifetime=4087], [size=0], [rc=204], [retryCount=0]
12-06 21:13:58.744 I/jellyfin-apiclient(21374): Response received from: http://192.168.1.200:8096/Sessions/Playing/Progress
12-06 21:13:59.086 D/CustomPlaybackOverlayFragment(21374): clearing subtitle event: 255 (pos 1408796 - event end 1408448)
12-06 21:13:59.912 D/ACRA    (21374): onActivityPaused class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:13:59.943 D/SurfaceUtils(21374): connecting to surface 0xbea9f408, reason connectToSurface
12-06 21:13:59.943 I/MediaCodec(21374): [OMX.amlogic.avc.decoder.awesome2] setting surface generation to 21886992
12-06 21:13:59.943 D/SurfaceUtils(21374): disconnecting from surface 0xbea9f408, reason connectToSurface(reconnect)
12-06 21:13:59.943 D/SurfaceUtils(21374): connecting to surface 0xbea9f408, reason connectToSurface(reconnect)
12-06 21:13:59.946 D/SurfaceUtils(21374): disconnecting from surface 0xbea9f408, reason setNativeWindowSizeFormatAndUsage
12-06 21:13:59.946 D/SurfaceUtils(21374): connecting to surface 0xbea9f408, reason setNativeWindowSizeFormatAndUsage
12-06 21:13:59.946 D/SurfaceUtils(21374): set up nativeWindow 0xbea9f408 for 1920x1080, color 0x11, rotation 0, usage 0x402900
12-06 21:13:59.946 D/SurfaceUtils(21374): disconnecting from surface 0xf1a69808, reason disconnectFromSurface
12-06 21:13:59.946 E/Surface (21374): freeAllBuffers: 4 buffers were freed while being dequeued!
12-06 21:13:59.949 D/ACRA    (21374): onActivityStopped class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:13:59.951 D/CustomPlaybackOverlayFragment(21374): Stopping!
12-06 21:13:59.952 D/CustomPlaybackOverlayFragment(21374): this fragment belongs to the current session, ending it
12-06 21:13:59.953 D/PlaybackController(21374): stop called at 1409301
12-06 21:13:59.991 D/MediaCodec(21374): keep callback message for reclaim
12-06 21:14:00.000 D/SurfaceUtils(21374): connecting to surface 0xc3699c08, reason connectToSurface
12-06 21:14:00.000 I/MediaCodec(21374): [OMX.amlogic.avc.decoder.awesome2] setting surface generation to 21886993
12-06 21:14:00.000 D/SurfaceUtils(21374): disconnecting from surface 0xc3699c08, reason connectToSurface(reconnect)
12-06 21:14:00.001 D/SurfaceUtils(21374): connecting to surface 0xc3699c08, reason connectToSurface(reconnect)
12-06 21:14:00.003 D/SurfaceUtils(21374): disconnecting from surface 0xc3699c08, reason setNativeWindowSizeFormatAndUsage
12-06 21:14:00.003 D/SurfaceUtils(21374): connecting to surface 0xc3699c08, reason setNativeWindowSizeFormatAndUsage
12-06 21:14:00.003 D/SurfaceUtils(21374): set up nativeWindow 0xc3699c08 for 1920x1080, color 0x11, rotation 0, usage 0x402900
12-06 21:14:00.003 D/SurfaceUtils(21374): disconnecting from surface 0xbea9f408, reason disconnectFromSurface
12-06 21:14:00.003 E/Surface (21374): freeAllBuffers: 4 buffers were freed while being dequeued!
12-06 21:14:00.077 D/SurfaceUtils(21374): disconnecting from surface 0xc3699c08, reason disconnectFromSurface
12-06 21:14:00.109 I/jellyfin-apiclient(21374): ReportPlaybackStopped: Item 881fc78da36c7a00c6c606b71e4ecc10, Ticks: 14093010000
12-06 21:14:00.110 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Sessions/Playing/Stopped
12-06 21:14:00.116 I/ExoPlayerImpl(21374): Release cb5e764 [ExoPlayerLib/2.18.1] [sabrina, Chromecast, Google, 31] [goog.exo.core, goog.exo.ui, goog.exo.exoplayer, goog.exo.decoder, goog.exo.datasource, goog.exo.extractor, goog.exo.ffmpeg]
12-06 21:14:00.155 D/ACRA    (21374): onActivitySaveInstanceState class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:14:00.158 D/ACRA    (21374): onActivityDestroyed class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:14:00.200 I/AppThemeCallbacks(21374): Applying theme: DARK
12-06 21:14:00.205 D/ACRA    (21374): onActivityCreated class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:14:00.326 D/ACRA    (21374): onActivityDestroyed class org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity
12-06 21:14:00.344 D/MediaManager(21374): Removed event listener.  Total listeners: 2
12-06 21:14:00.356 I/AppThemeCallbacks(21374): Applying theme: DARK
12-06 21:14:00.359 D/ACRA    (21374): onActivityCreated class org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity
12-06 21:14:00.375 E/ThemeUtils(21374): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
12-06 21:14:00.388 D/jellyfin-apiclient(21374): Adding request to queue: http://192.168.1.200:8096/Users/a0a85437-2854-441e-a91a-c99d5501acbb/Items/881fc78da36c7a00c6c606b71e4ecc10?format=json
12-06 21:14:00.389 D/ACRA    (21374): onActivityStarted class org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity
12-06 21:14:00.406 D/ACRA    (21374): onActivityResumed class org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity
12-06 21:14:00.408 D/BackgroundServiceFragment(21374): Restoring background drawable
12-06 21:14:00.427 D/MediaManager(21374): Added event listener.  Total listeners: 3
12-06 21:14:00.435 W/RecyclerView(21374): No adapter attached; skipping layout
12-06 21:14:00.435 W/RecyclerView(21374): No adapter attached; skipping layout
12-06 21:14:00.895 D/ACRA    (21374): onActivityDestroyed class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
12-06 21:14:01.158 D/FullDetailsActivity(21374): current time 1670382841157 last playback event time 1670382840110 last refresh time 1670382840388
12-06 21:14:01.158 D/AndroidRuntime(21374): Shutting down VM
--------- beginning of crash
12-06 21:14:01.159 E/AndroidRuntime(21374): FATAL EXCEPTION: main
12-06 21:14:01.159 E/AndroidRuntime(21374): Process: org.jellyfin.androidtv, PID: 21374
12-06 21:14:01.159 E/AndroidRuntime(21374): java.lang.NullPointerException: Attempt to invoke virtual method 'org.jellyfin.apiclient.model.dto.BaseItemType org.jellyfin.apiclient.model.dto.BaseItemDto.getBaseItemType()' on a null object reference
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity$2.run(FullDetailsActivity.java:256)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at android.os.Handler.handleCallback(Handler.java:938)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at android.os.Handler.dispatchMessage(Handler.java:99)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at android.os.Looper.loopOnce(Looper.java:201)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at android.os.Looper.loop(Looper.java:288)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at java.lang.reflect.Method.invoke(Native Method)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
12-06 21:14:01.159 E/AndroidRuntime(21374): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
12-06 21:14:01.161 E/ACRA    (21374): ACRA caught a NullPointerException for org.jellyfin.androidtv
12-06 21:14:01.161 E/ACRA    (21374): java.lang.NullPointerException: Attempt to invoke virtual method 'org.jellyfin.apiclient.model.dto.BaseItemType org.jellyfin.apiclient.model.dto.BaseItemDto.getBaseItemType()' on a null object reference
12-06 21:14:01.161 E/ACRA    (21374): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsActivity$2.run(FullDetailsActivity.java:256)
12-06 21:14:01.161 E/ACRA    (21374): 	at android.os.Handler.handleCallback(Handler.java:938)
12-06 21:14:01.161 E/ACRA    (21374): 	at android.os.Handler.dispatchMessage(Handler.java:99)
12-06 21:14:01.161 E/ACRA    (21374): 	at android.os.Looper.loopOnce(Looper.java:201)
12-06 21:14:01.161 E/ACRA    (21374): 	at android.os.Looper.loop(Looper.java:288)
12-06 21:14:01.161 E/ACRA    (21374): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
12-06 21:14:01.161 E/ACRA    (21374): 	at java.lang.reflect.Method.invoke(Native Method)
12-06 21:14:01.161 E/ACRA    (21374): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
12-06 21:14:01.161 E/ACRA    (21374): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
12-06 21:14:01.161 D/ACRA    (21374): Building report
12-06 21:14:01.168 D/ACRA    (21374): Starting collectors with priority FIRST
12-06 21:14:01.169 D/ACRA    (21374): Calling collector org.acra.collector.DropBoxCollector
12-06 21:14:01.169 D/ACRA    (21374): Calling collector org.acra.collector.LogCatCollector
12-06 21:14:01.170 D/ACRA    (21374): Collector org.acra.collector.DropBoxCollector completed
12-06 21:14:01.170 D/ACRA    (21374): Calling collector org.acra.collector.StacktraceCollector
12-06 21:14:01.172 D/ACRA    (21374): Collector org.acra.collector.StacktraceCollector completed
12-06 21:14:01.177 D/ACRA    (21374): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.14.5` (`140599`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"sabrina","BOOTLOADER":"01.01.221014.042927","BRAND":"google","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"sabrina","DISPLAY":"STTE.220920.015.A1.9261725","FINGERPRINT":"google\/sabrina_prod_stable\/sabrina:12\/STTE.220920.015.A1\/9261725:user\/release-keys","HARDWARE":"amlogic","HOST":"abfarm-release-2004-0103","ID":"STTE.220920.015.A1","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Google","MODEL":"Chromecast","ODM_SKU":"US","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"sabrina_prod_stable","RADIO":"unknown","SKU":"US","SOC_MANUFACTURER":"Amlogic","SOC_MODEL":"AMLS905D3","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1667789403000,"TYPE":"user","UNKNOWN":"unknown","USER":"android-build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"9261725","MEDIA_PERFORMANCE_CLASS":0,"PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"12","RELEASE_OR_CODENAME":"12","SDK":"31","SDK_INT":31,"SECURITY_PATCH":"2022-10-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":140599,"VERSION_NAME":"0.14.5"}
```  

### Device information

***Android version***: `12`  
***Device brand***: `google`  
***Device product***: `sabrina_prod_stable`  
***Device model***: `Chromecast`  

### Crash information

***Start time***: `2022-12-04T22:20:37.190-06:00`  
***Crash time***: `2022-12-06T21:14:01.334-06:00`  

### FFmpeg logs

```shell
NA - Direct Play

### Jellyfin Server Logs from around the crash timestamp
[2022-12-06 21:05:05.564 -06:00] [INF] [20] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Android TV" "0.14.5" playing <Title>. Stopped at "1009334" ms
[2022-12-06 21:05:28.523 -06:00] [INF] [53] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 1 WebSockets.
[2022-12-06 21:14:01.118 -06:00] [INF] [78] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Android TV" "0.14.5" playing <Title>. Stopped at "1409301" ms
[2022-12-06 21:14:06.597 -06:00] [WRN] [55] Trakt.ServerMediator: Received playback progress from user "" but initial state was never set - setting it now!
[2022-12-06 21:14:14.517 -06:00] [INF] [91] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.3.112" request
```

### Media info of the file

```shell
ffprobe version 4.3.4-0+deb11u1+rpt3 Copyright (c) 2007-2021 the FFmpeg developers
  built with gcc 10 (Debian 10.2.1-6)
  configuration: --prefix=/usr --extra-version=0+deb11u1+rpt3 --toolchain=hardened --incdir=/usr/include/aarch64-linux-gnu --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --disable-mmal --enable-neon --enable-v4l2-request --enable-libudev --enable-epoxy --enable-sand --libdir=/usr/lib/aarch64-linux-gnu --arch=arm64 --enable-pocketsphinx --enable-libdc1394 --enable-libdrm --enable-vout-drm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
Input #0, matroska,webm, from <TITLE>:
  Metadata:
    encoder         : libebml v1.4.0 + libmatroska v1.6.1
  Duration: 00:56:41.07, start: 0.000000, bitrate: 8590 kb/s
    Stream #0:0: Video: h264 (High), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 23.98 fps, 23.98 tbr, 1k tbn, 47.95 tbc (default)
    Stream #0:1(eng): Audio: ac3, 48000 Hz, 5.1(side), fltp, 384 kb/s (default)
    Stream #0:2(eng): Subtitle: subrip
    Metadata:
      title           : English [SDH]
    Stream #0:3(spa): Subtitle: subrip
    Metadata:
      title           : Latin America Spanish [SDH]
```

### Application version

0.14.5

### Device information

Chromecast for Google TV 2022

### Android version

12

### Jellyfin server version

10.8.8

**Comment by nielsvanvelzen** [comment]
This might be fixed in the 0.15 betas. Can you try that and update the logs if it still crashes?

**Comment by manuel-rw** [comment]
I had a this issue too, when I tried to start playback for anything - app freeze and then crash.
I can confirm that beta 5 solved the issue for me.
